monkestation/code/modules/viking/viking_axes.dm

```diff
@@ -0,0 +1,93 @@
+// fear the øx
+// øx means axe for the uncultured
+
+
+
+/obj/item/melee/viking
+	icon = 'monkestation/icons/viking/viking_items.dmi'
+	lefthand_file = 'monkestation/icons/viking/axes_lefthand.dmi'
+	righthand_file = 'monkestation/icons/viking/axes_righthand.dmi'
+	worn_icon = 'monkestation/icons/viking/viking_armor.dmi'
+
+/obj/item/melee/viking/tenja
+	name = "boarding axe"
+	icon_state = "hand_axe_item"
+	lefthand_file = "hand_axe_inhand_L"
+	righthand_file = "hand_axe_inhand_R"
```

Comment:
Don't change the inhand_file. This is making it look for a file called "hand_axe_inhand_L" not the actual sprites within. It won't work if you do it like this.